snet_cli/utils.py

```diff
@@ -123,6 +127,16 @@ def walk_imports(entry_path):
     return seen_paths
 
 
+def get_contract_dict(contract_path, contract_name):
```

Comment:
IMO either hard-code `contract_path` in the body of this fn (since only one path is used anywhere), or make `contract_name` positional and give `contract_path` a default of the one path used everywhere else. Then we can remove the repetitive `Path(__file__).absolute().parent.joinpath("resources", "contracts")` from everywhere else in code. Also, for reference, this wasn't going to be `contract_path` per our Slack convo, I'm having a naming meltdown